sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-examples.adoc

```diff
@@ -1,161 +1,538 @@
+
+. In Studio, select *File > New > Mule Project*.
+. In *Project Name*, enter a name for your Mule project. +
+This example uses the name `successfactors-operations`. 
+. Click *Finish*.
+
+=== Specify Your Credentials
+
+. In *Package Explorer*, under `/src/main/resources/`, create a file named `mule-app.properties`. 
+. Specify your credentials for connecting to SuccessFactors, for example:
++
+----
+config.companyId=<your-company-id>
+config.userName=<successfactors-username>
+config.password=<successfactors-password>
+----
++ 
+. Save the file. 
+
+[[configure-global-element]]
+=== Configure a Global Element for the Connector
+
+Configure a global element to provide the authentication credentials that the connector requires to access the target SAP SuccessFactors system.
+
+. At the base of the Studio canvas, click *Global Elements*. 
+. Select *SAP SuccessFactors Config* and click *Edit*. 
+. In the SAP SuccessFactors Configuration window, enter the connection credentials as follows:
+* *Company ID*: `${config.companyId}`
+* *User Name*: `${config.userName}`
+* *Password*:`${config.password}`
+. From the *Data Center's Endpoint URL* drop-down, select the SuccessFactor's Web API URL. 
++
+The following image shows an example of configuring Basic authentication:
++
+.Basic authentication fields
+image::sap-successfactors-basic-config.png
++
+. Click *OK*. 
+. Click *Test Connection* to verify the connection to SAP SuccessFactors.
++
+A success message appears.
+
+[[create-query-flow]]
+== Create the Query Operation Flow 
+
+Create a flow that uses the *Query* operation to perform queries over entities of a specified type:
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify `/list` for the *Path* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of HTTP *Listener* in the app.
+.. On the *General* tab, specify connection information for the connector.
+.. Click *Test Connection* to confirm that Mule can connect with the specified server.
+.. Click *OK*.
+. In the *Mule Palette* view, select *SAP SuccessFactors*, then select the *Query* operation and drag it on to the Studio canvas to the right of the *Listener* operation. 
+. In the left pane of the *Mule Palette*, click *Favorites*.
+. Select *Transform Message* and drag it on to the Studio canvas to the right of the *Query* operation. +
+The *Transform Message* component transforms the data structure and format to produce the output the connector expects, in this case, `application/json`:
++
+----
+%dw 2.0
+output application/json
+---
+payload
+
+----
++ 
+. Add a *Logger* component to the end of the flow that takes a `payload` (or `#[payload]`) as the message.
```

Comment:
Misplaced modifier:  It should be this: 
Add a *Logger* component that takes a `payload` (or `#[payload]`) as the message to the end of the flow.
.